x-pack/filebeat/module/googlecloud/vpcflow/ingest/pipeline.yml

```diff
@@ -0,0 +1,51 @@
+description: Pipeline for Google Cloud VPC Flow Logs
+
+processors:
+  # IP Geolocation Lookup
+  - geoip:
+      field: source.ip
+      target_field: source.geo
+      ignore_missing: true
+  - geoip:
+      field: destination.ip
+      target_field: destination.geo
+      ignore_missing: true
+
+  # IP Autonomous System (AS) Lookup
+  - geoip:
+      database_file: GeoLite2-ASN.mmdb
```

Comment:
Do these come by default with the maxmind installation? Will it degrade somewhat gracefully if the files are not found?

Reply:
They come with Elasticsearch by default. ([source](https://www.elastic.co/guide/en/elasticsearch/reference/7.2/geoip-processor.html#using-ingest-geoip))

> The database filename in the geoip config directory. The ingest-geoip module ships with the GeoLite2-City.mmdb, GeoLite2-Country.mmdb and GeoLite2-ASN.mmdb files.

I assume the pipeline would fail if someone removed them.